src/level.ts

```diff
@@ -155,105 +77,109 @@ export function level(useResources = true) {
       buy(1, $item`bowl of potpourri`);
     }
     // Chateau rest
-    while (getPropertyInt('timesRested') < totalFreeRests()) {
-      visitUrl('place.php?whichplace=chateau&action=chateau_restbox');
+    while (get("timesRested") < totalFreeRests()) {
+      visitUrl("place.php?whichplace=chateau&action=chateau_restbox");
     }
   }
 
-  cliExecute('breakfast');
+  cliExecute("breakfast");
 
   // LOV Tunnel
-  if (get('loveTunnelAvailable') && !getPropertyBoolean('_loveTunnelUsed') && useResources) {
+  if (get("loveTunnelAvailable") && !get("_loveTunnelUsed") && useResources) {
     useFamiliar($familiar`Hovering Sombrero`);
     const macro = Macro.if_(
-      'monstername LOV Enforcer',
-      Macro.while_('!pastround 20 && !hpbelow 200', Macro.attack().repeat()).kill()
+      "monstername LOV Enforcer",
+      Macro.while_("!pastround 20 && !hpbelow 200", Macro.attack().repeat()).kill()
     )
-      .if_('monstername LOV Engineer', Macro.skill($skill`Saucegeyser`).repeat())
+      .if_("monstername LOV Engineer", Macro.skill($skill`Saucegeyser`).repeat())
```

Comment:
weapon of the pastalord is better (candyblast and strignozzi serpent if we're being dicks) because of elemental res

Reply:
everyone has saucegeyser permed though